File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--creating-a-web-part-for-sharepoint_3.cs
        [Personalizable(PersonalizationScope.Shared), WebBrowsable(true),
        WebDisplayName("Path to Employee Data File"),
        WebDescription("Location of the XML file that contains employee data")]
        public string DataFilePath
        {
            get
            {
                return xmlFilePath;
            }
            set
            {
                xmlFilePath = value;
            }
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--creating-a-web-part-for-sharepoint_3.vb
    <Personalizable(PersonalizationScope.[Shared]), _
        WebBrowsable(True), WebDisplayName("Path to Employee Data File"), _
        WebDescription("Location of the XML file that contains employee data")> _
    Public Property DataFilePath() As String
        Get
            Return xmlFilePath
        End Get
        Set(ByVal value As String)
            xmlFilePath = value
        End Set
    End Property